### PR TITLE
[ES] Update tests for homeassistant_HassTurnOn.yaml

### PR DESCRIPTION
**Pull request by khymmera**
Added some alternative commands

### CODE REVIEW

File: tests/es/homeassistant_HassTurnOn.yaml [hunk 1]
```diff
@@ -7,3 +7,15 @@ tests:
       name: "HassTurnOn"
       slots:
         name: "fan.ventilador_techo"
+  - sentences:
+      - "abre la cortina izquierda"
```
Comment (balloob) [inline tests/es/homeassistant_HassTurnOn.yaml:11]: Cover specific sentences should be added to `cover_HassTurnOn.yaml`

Comment (khymmera) [reply tests/es/homeassistant_HassTurnOn.yaml]: Removed sentence.

File: tests/es/homeassistant_HassTurnOn.yaml [hunk 2]
```diff
@@ -7,3 +7,9 @@ tests:
       name: "HassTurnOn"
       slots:
         name: "fan.ventilador_techo"
+  - sentences:
```
Comment (cibernox) [inline tests/es/homeassistant_HassTurnOn.yaml:10]: While it's not incorrect,do we ever refer to the switch when turning on/off something? I always ask to turn on/off the light (or whatever) controlled by the switch, but never the switch itself. Perhaps there's an use case I'm not aware of.

Comment (davefx) [reply tests/es/homeassistant_HassTurnOn.yaml:10]: Sometimes you could have a generic switch in a room, where you connect several devices. 
In that case, you would use this sentence.

**Comment by davefx** [comment]
In order to merge the PR, tests must pass

**Pull request merged**